src/openrct2/ride/RideRatings.h

```diff
@@ -19,7 +19,7 @@ using track_type_t = uint16_t;
 // Convenience function for writing ride ratings. The result is a 16 bit signed
 // integer. To create the ride rating 3.65 type RIDE_RATING(3,65)
 #define RIDE_RATING(whole, fraction) FIXED_2DP(whole, fraction)
-#define RIDE_RATING_UNDEFINED static_cast<ride_rating>(static_cast<uint16_t>(0xFFFF))
+constexpr ride_rating kRideRatingUndefined = static_cast<ride_rating>(static_cast<uint16_t>(0xFFFF));
```

Comment:
We can probably drop one of these casts here.

Reply:
You can try changing `static_cast<uint16_t>(0xFFFF)` to `uint16_t(0xFFFF)` to get rid of one cast, or alternatively append the needed postfixes (`u` and another one).

Reply:
`uint16_t(0xFFFF)` is still a cast, just of the C-style function variety. Adding a prefix (i.e. `0xFFFFU`) would therefore be preferable.

Reply:
All good suggestions, I did remove all `static_cast` and added the `u` postfix, seems to work. Am I missing anything here?